app/controllers/EventsController.scala

```diff
@@ -19,30 +19,30 @@ package controllers
 import model.persisted.eventschedules.{ EventType, VenueType }
 import play.api.Logger
 import play.api.libs.json.Json
-import play.api.mvc.Action
+import play.api.mvc.{ Action, AnyContent }
 import repositories.events.EventsRepository
-import services.assessmentcentre._
+import services.events.EventsParsingService
 import uk.gov.hmrc.play.microservice.controller.BaseController
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
 object EventsController extends EventsController {
   override val assessmentEventsRepository: EventsRepository = repositories.eventsRepository
-  override val assessmentCenterParsingService: AssessmentCentreParsingService = AssessmentCentreParsingService
+  override val assessmentCenterParsingService: EventsParsingService = EventsParsingService
```

Comment:
Minor point/suggestion (i.e. no need to change it you don't want to): "override" keyword is not required for an abstract field (in parent).